src/cm.css

```diff
@@ -8,6 +8,13 @@
   --text-color: #010101;
 }
 
+/** Global constants */
+.root, .root > * {
```

Comment:
Does this work with Safari 12.1?

Reply:
Yes, tested on Safari 12.1. Note this is a simple class selector rather than the actual `:root` selector.

<img width="1792" alt="Screen Shot 2023-01-04 at 10 03 40 AM" src="https://user-images.githubusercontent.com/7354176/210620434-f1f125f1-69fa-4552-9d73-57b041514329.png">